src/transcoder/transcoderoptionsflac.cpp

```diff
@@ -37,9 +39,30 @@ void TranscoderOptionsFlac::Load() {
   ui_->quality->setValue(s.value("quality", 5).toInt());
 }
 
+void TranscoderOptionsFlac::ValueChanged(int value) {
+  if (!IsInStreamingSubset(value)) {
+    ui_->info->setStyleSheet("QLabel { color : red }");
+    ui_->info->setText(
+        tr("Warning: This compression level is outside of the streamable "
```

Comment:
You might want to expand on what the practical implications of that are.

Reply:
Added more info, but open to other suggestions.